### PR TITLE
[PREVIEW] SL-1751: add possibility to create listing request note

### PR DESCRIPTION
**Pull request by r-kruk-kainos**

### CODE REVIEW

File: src/app/hearing/components/view-hearing/view-hearing.component.ts [hunk 1]
```diff
@@ -125,4 +136,12 @@ export class ViewHearingComponent implements OnInit {
   private fetchHearing() {
```
Comment (radoslawlandowskikainos) [inline src/app/hearing/components/view-hearing/view-hearing.component.ts]: private methods below publics

File: src/app/hearing/components/view-hearing/view-hearing.component.ts [hunk 2]
```diff
@@ -125,4 +136,12 @@ export class ViewHearingComponent implements OnInit {
...
     this.hearingService.getById(this.hearingId);
```
Comment (radoslawlandowskikainos) [inline src/app/hearing/components/view-hearing/view-hearing.component.ts]: Don't you have to subscribe to this function to make it work?

File: src/app/hearing/components/create-hearing-note/create-hearing-note.component.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<div fxLayout="row" fxLayoutAlign="start start">
```
Comment (zielinskikamil) [inline src/app/hearing/components/create-hearing-note/create-hearing-note.component.html]: @r-kruk-kainos What I know this **fxLayoutAlign="start start"** was causing UI issue on IE. @andrewtaylorkainos plz correct me if I'm wrong.

@r-kruk-kainos  can you make sure that its display ok on IE?

File: src/app/hearing/components/view-hearing/view-hearing.component.ts [hunk 4]
```diff
@@ -20,25 +25,53 @@ export class ViewHearingComponent implements OnInit {
...
     this.hearingId = this.route.snapshot.paramMap.get('id');
+    this.note = this.listingCreateNotesConfiguration.getOrCreateNote([], NoteType.LISTING_NOTE);
     this.hearingService.hearings
       .map(hearings => hearings.find(h => h.id === this.hearingId))
       .subscribe(hearing => this.hearing = hearing);
 
     this.fetchHearing();
   }
 
+  private confirmationDialogClosed = (confirmed: boolean) => {
```
Comment (zielinskikamil) [inline src/app/hearing/components/view-hearing/view-hearing.component.ts]: is there any reason why you moved private methods above internal one?

File: src/app/hearing/components/create-hearing-note/create-hearing-note.component.html (hunk 3 above)
Comment (r-kruk-kainos) [reply src/app/hearing/components/create-hearing-note/create-hearing-note.component.html]: I've checked it on IE and it was displaying ok.

**Pull request merged**